src/AppInstallerCommonCore/ExperimentalFeature.cpp

```diff
@@ -73,6 +75,8 @@ namespace AppInstaller::Settings
             return ExperimentalFeature{ "Show Dependencies Information", "dependencies", "https://aka.ms/winget-settings", Feature::Dependencies };
         case Feature::DirectMSI:
             return ExperimentalFeature{ "Direct MSI Installation", "directMSI", "https://aka.ms/winget-settings", Feature::DirectMSI };
+        case Feature::PortableInstall:
+            return ExperimentalFeature{ "Portable Installation", "portableInstall", "https://aka.ms/winget-settings", Feature::PortableInstall };
```

Comment:
>https://aka.ms/winget-settings [](http://example.com/codeflow?start=85&length=30)

remember to update the settings md doc

Reply:
Updated settings.md doc for this experimental feature